ecs-init/engine/engine.go

```diff
@@ -143,9 +143,12 @@ func (e *Engine) PreStart() error {
 		return engineError("could not create route to the credentials proxy", err)
 	}
 	// Add the EBS Task Attach host mount point
-	err = os.MkdirAll(config.MountDirectoryEBS(), mountFilePermission)
-	if err != nil {
-		return engineError("could not create EBS mount directory", err)
+	// Skip for External, EBS Task Attach is not supported for External launch type
+	if !config.RunningInExternal() {
```

Comment:
This bug affects EC2 launch type too doesn't it? I am wondering if we should just log the failure and continue. Can we disable the EBS capability if we fail to create this directory?

Reply:
Also, when running in external we're going to advertise the capability anyways here? even though we failed to make the mnt directory?